### DISCUSSION TITLE
Docker Disk Usage Issue: "docker_data.vhdx" Growing Too Large

### DISCUSSION THREAD

**Discussion by cheonmux** [danny-avila/LibreChat#4916] [Q&A]
Hi everyone,

I'm using Libre Chat on Docker under Windows 11, and it's working great. However, I'm running into an issue where the "docker_data.vhdx" file in my Docker folder is growing very large. 

My main concern is: If I manually delete the "docker_data.vhdx" file, will it erase all my past conversations in Libre Chat? Is it safe to remove it, or will I lose my data?

The file is currently around 30GB, which seems excessive since I haven't had that many conversations. It doesn't make sense to me why it's taking up so much space.

I'm wondering if there's a way to limit the size of this file or reduce its size without losing my chat history.

Any advice or insights would be greatly appreciated! Thanks in advance.

**Comment by danny-avila** [comment]
Hi, thanks for asking. That is specific to docker, and has nothing to do with LC conversations.

While LibreChat is running, you can run `docker system prune -a` which should help clear up any unused docker images.

More info (generated by AI): https://demo.librechat.ai/share/FfteFI8cBxNy2hKVZb3qE
